tests/sdk/v2/api/test_wf_run.py

```diff
@@ -592,6 +592,102 @@ def test_get_tasks_from_started_workflow(run):
                 assert task._wf_def == run._wf_def
                 assert task.task_invocation_id in wf_def_model.task_invocations
 
+        class TestFiltering:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Can we test with multiple conditions? e.g. filtering with a state and a task ID or function name?

Reply:
Good idea, I've added that to the unit testing for the filter methods in [662e649](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/188/commits/662e649e73384213a77de52143fe7f618d4924b4)